at_lookup/lib/src/connection/outbound_message_listener.dart

```diff
@@ -92,33 +94,46 @@ class OutboundMessageListener {
   }
 
   /// Reads the response sent by remote socket from the queue.
-  /// If there is no message in queue after [maxWaitMilliSeconds], return null. Defaults to 3 seconds.
-  Future<String> read({int maxWaitMilliSeconds = 10000}) async {
-    return _read(maxWaitMillis: maxWaitMilliSeconds);
-  }
-
-  Future<String> _read({int maxWaitMillis = 10000, int retryCount = 1}) async {
+  /// If there is no message in queue after [maxWaitMilliSeconds], return null. Defaults to 90 seconds.
+  Future<String> read(
```

Comment:
please add comment for transientWaitTimeMilllis parameter

Reply:
done